### PR TITLE
fix: prevent empty source list render

### PR DESCRIPTION
**Pull request by rebelchris**
### Describe what this PR does
- Prevent empty source render when done loading but sources still empty

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-{number} #done

### CODE REVIEW

File: packages/shared/src/components/search/SearchSourceList.tsx [hunk 1]
```diff
@@ -29,6 +29,10 @@ export const SearchSourceList = ({
     limit: 3,
   });
 
+  if (!isLoading && !sources) {
+    return null;
+  }
```
Comment (capJavert) [inline packages/shared/src/components/search/SearchSourceList.tsx:34]: Should we check `sources?.length > 0`?

Comment (rebelchris) [reply packages/shared/src/components/search/SearchSourceList.tsx]: Good point, changed now.

**Review by capJavert — APPROVED** [review]

File: packages/shared/src/components/search/SearchSourceList.tsx [hunk 2]
```diff
@@ -29,6 +29,10 @@ export const SearchSourceList = ({
     limit: 3,
   });
 
+  if (!isLoading && sources?.length === 0) {
```
Comment (sshanzel) [inline packages/shared/src/components/search/SearchSourceList.tsx:32]: I think the condition should have been `!sources?.length`. Undefined or null would be false on this condition.

Comment (sshanzel) [reply packages/shared/src/components/search/SearchSourceList.tsx:32]: Nvm. I see the other PR now with the right condition.

**Pull request merged**